e2e/pkg/dump/dump.go

```diff
@@ -0,0 +1,62 @@
+package dump
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	yaml "sigs.k8s.io/yaml/goyaml.v3"
+
+	tcontext "github.com/konflux-workspaces/workspaces/e2e/pkg/context"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	workspacesv1alpha1 "github.com/konflux-workspaces/workspaces/operator/api/v1alpha1"
+)
+
+var resourcesToDump = func() []client.ObjectList {
+	return []client.ObjectList{
+		&workspacesv1alpha1.InternalWorkspaceList{},
+		&toolchainv1alpha1.UserSignupList{},
+		&toolchainv1alpha1.MasterUserRecordList{},
+		&toolchainv1alpha1.SpaceList{},
+		&toolchainv1alpha1.SpaceBindingList{},
+	}
+}
```

Comment:
```suggestion
This could probably be `const`:

const resourcesToDump []client.ObjectList = {
	&workspacesv1alpha1.InternalWorkspaceList{},
	&toolchainv1alpha1.UserSignupList{},
	&toolchainv1alpha1.MasterUserRecordList{},
	&toolchainv1alpha1.SpaceList{},
	&toolchainv1alpha1.SpaceBindingList{},
}
```